testgrid/cmd/configurator/main.go

```diff
@@ -324,11 +324,14 @@ func main() {
 		return
 	}
 
-	// Setup GCS client
+	// Set up GCS client if output is to GCS
 	var client *storage.Client
-	if opt.output != "" {
-		// Error returned if outputting to file; ignore for now
-		client, _ = gcs.ClientWithCreds(ctx, opt.creds)
+	if strings.HasPrefix(opt.output, "gs://") {
+		var err error
+		client, err = gcs.ClientWithCreds(ctx, opt.creds)
+		if err != nil {
+			log.Fatalf("FAIL: Credential error: %v", err)
```

Comment:
I don't think "credential error" is either necessarily accurate nor actually necessary here.

Reply:
That's fair. `Client Error`? Other suggestions?
I want _something_ to indicate that the error is occurring with the `gcs.ClientWithCreds` call, and not failing for some other reason.

Reply:
"failed to create gcs client" would be an accurate description of what actually failed here.